lib/wasi/src/syscalls/wasix/bus_subcall.rs

```diff
@@ -0,0 +1,26 @@
+use super::*;
+use crate::syscalls::*;
+
+// FIXME: remove , since it's no longer used.
```

Comment:
So this need to be removed? Shall I do it?

Reply:
Removed